### ISSUE TITLE
Question about training a model with value generation

### ISSUE THREAD

**Issue by fyinh** [rhythmcao/text2sql-lgesql#16]
I am so interested in your excellent work and follow your instructions to reproduce the experiment.
I am wondering if i could train a model with value gneration rather that a special token "value"?

**Comment by nixonjin** [comment]
自动回复：这里是金能征的邮箱，您的信件已被收入。This is Jin Nengzheng. Your letter have been received and will be disposed as soon as possible.

**Comment by nixonjin** [comment]
Yes, you can. You have to change the grammar file and rewrite the transition system.

**Comment by rhythmcao** [comment]
Working towards that. Codes will be released if everything is ok.

**Comment by eliotwalt** [comment]
Any news about that issue?

**Comment by nixonjin** [comment]
自动回复：这里是金能征的邮箱，您的信件已被收入。This is Jin Nengzheng. Your letter have been received and will be disposed as soon as possible.

**Comment by jc-ryan** [comment]
> Working towards that. Codes will be released if everything is ok.

how about the process？Thanks.

**Comment by dtdxdydz** [comment]
Any news on the issue?

**Comment by Elienvalleau** [comment]
> Yes, you can. You have to change the grammar file and rewrite the transition system.

hello,
what are the changes to make to the grammar file and the transition system?
